### PR TITLE
Only reload single file when in the static path

### LINKED ISSUE #241: reload stop working if there's a database file changed.
```
[03:01:31] expected str, bytes or os.PathLike object, not NoneType
Traceback (most recent call last):
  File "c:\program files\python37\lib\site-packages\aiohttp_devtools\runserver\watch.py", line 69, in _run
    await src_reload(self._app, changes.pop()[1])
  File "c:\program files\python37\lib\site-packages\aiohttp_devtools\runserver\serve.py", line 161, in src_reload
    path = str(Path(app['static_url']) / Path(path).relative_to(app['static_path']))
  File "c:\program files\python37\lib\pathlib.py", line 866, in relative_to
    to_drv, to_root, to_parts = self._parse_args(other)
  File "c:\program files\python37\lib\pathlib.py", line 642, in _parse_args
    a = os.fspath(a)
TypeError: expected str, bytes or os.PathLike object, not NoneType
ERROR:adev.server.dft:expected str, bytes or os.PathLike object, not NoneType
Traceback (most recent call last):
  File "c:\program files\python37\lib\site-packages\aiohttp_devtools\runserver\watch.py", line 69, in _run
    await src_reload(self._app, changes.pop()[1])
  File "c:\program files\python37\lib\site-packages\aiohttp_devtools\runserver\serve.py", line 161, in src_reload
    path = str(Path(app['static_url']) / Path(path).relative_to(app['static_path']))
  File "c:\program files\python37\lib\pathlib.py", line 866, in relative_to
    to_drv, to_root, to_parts = self._parse_args(other)
  File "c:\program files\python37\lib\pathlib.py", line 642, in _parse_args
    a = os.fspath(a)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

```
changes in watch.py =  {(<Change.modified: 2>, 'C:\\Users\\a\\Desktop\\a.db')}
path in pathlib= C:\Users\a\Desktop\a.db
```

### LINKED ISSUE #256: .goutputstream does not start with static
* **aiohttp-devtools version:** latest
* **aiohttp version:** latest
* **python version:** 3.6
* **Platform:** Ubuntu 18.04

### Issue Summary

I'm frequently getting this error when the server tries to reload:

```
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/aiohttp_devtools/runserver/watch.py", line 68, in _run
    await src_reload(self._app, changes.pop()[1])
  File "/home/ubuntu/.local/lib/python3.6/site-packages/aiohttp_devtools/runserver/serve.py", line 160, in src_reload
    path = str(Path(app['static_url']) / Path(path).relative_to(app['static_path']))
  File "/usr/lib/python3.6/pathlib.py", line 874, in relative_to
    .format(str(self), str(formatted)))
ValueError: '/home/ubuntu/hacking/teslaclub/teslaclub_app/.goutputstream-AKHYI0' does not start with '/home/ubuntu/hacking/teslaclub/teslaclub_app/static'
```

### Steps to reproduce

Not sure, difficult to reproduce reliably.

### PR DESCRIPTION
**Pull request by Dreamsorcerer**
Fixes #256 
Probably fixes #241

### CODE REVIEW

File: aiohttp_devtools/runserver/watch.py [hunk 1]
```diff
@@ -53,19 +53,21 @@ async def _run(self, live_checks=20):
         try:
             self._start_dev_server()
 
+            static_path = str(self._app['static_path'])
             async for changes in self._awatch:
                 self._reloads += 1
+                is_static = all(str(c[1]).startswith(static_path) for c in changes)
```
Comment (samuelcolvin) [inline aiohttp_devtools/runserver/watch.py:59]: this doesn't need to be calculated until it's used, maybe move it inline, (perhaps via a separate function).

File: tests/test_runserver_watch.py [hunk 2]
```diff
@@ -76,6 +78,7 @@ async def test_python_no_server(loop, mocker):
     app_task._start_dev_server = MagicMock()
     app_task._stop_dev_server = MagicMock()
     app = Application()
+    app["static_path"] = "/path/to/"
```
Comment (samuelcolvin) [inline tests/test_runserver_watch.py:81]: single quotes please.

File: aiohttp_devtools/runserver/watch.py (hunk 1 above)
Comment (Dreamsorcerer) [reply aiohttp_devtools/runserver/watch.py]: Is a lambda fine? Just trying to keep the `is_static` name for readability.

Comment (Dreamsorcerer) [reply aiohttp_devtools/runserver/watch.py]: OK, linter says no.

**Comment by Dreamsorcerer** [comment]
@samuelcolvin Good to merge?

**Comment by samuelcolvin** [comment]
Thanks, sorry I've been so bad at replying.

**Comment by Dreamsorcerer** [comment]
No problem, figured you're probably dealing with pydantic and fastapi at the moment, so I don't mind taking over maintenance.

**Pull request merged**